nibabel/ecat.py

```diff
@@ -611,8 +611,8 @@ def _get_oriented_data(self, raw_data, orientation=None):
         elif orientation == 'radiological':
             orientation = patient_orient_radiological[0]
         else:
-            raise ValueError('orientation should be None,\
-                neurological or radiological')
+            raise ValueError('orientation should be None, '
```

Comment:
Might be cleaner just to put it on one line. Does that pass `flake8`?

Reply:
Indeed, let's try.